dags/process_elife_zip_dag.py

```diff
@@ -17,11 +17,13 @@
 from airflow.utils import timezone
 from lxml import etree
 from lxml.builder import ElementMaker
+from PIL import Image
 
 from aws import get_aws_connection
 from task_helpers import get_previous_task_name
 
-ALLOWED_EXTENSIONS = {'.jpg', '.jpeg', '.pdf', '.xml'}
+ALLOWED_EXTENSIONS = {'.jpg', '.jpeg', '.pdf', '.xml'}  # only upload these file types
+IMAGE_EXTENSIONS = {'.jpg', '.jpeg', '.png', '.tif', '.tiff'}
```

Comment:
seems reasonable so far as source images, in the end eLife uses only `.tif` for example